.github/workflows/docs.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - 'main'
-      - 'develop'
+      - 'precomp'
```

Comment:
Put this back to `develop` (we want that branch to be the develop branch for the docs).